autoagora/config.py

```diff
@@ -179,6 +179,17 @@ def init_config(argv: Optional[Sequence[str]] = None):
         default=3600,
         help="(Seconds) Interval between rebuilds of the relative query costs models.",
     )
+    #
+    # Manual agora entry values
+    #
+    argparser.add_argument(
+        "--aa-app-manual-entry-path",
+        env_var="AA_APP_MANUAL_ENTRY_PATH",
```

Comment:
```suggestion
        "--manual-entry-path",
        env_var="MANUAL_ENTRY_PATH",
```
We don't use a prefix on the other arguments.
A prefix may indeed be useful, but only for the environment variables. For this, it seems we could use `auto_env_var_prefix` in [`ArgumentParser`](https://bw2.github.io/ConfigArgParse/configargparse.ArgumentParser.html#__init__), which also removes the need for us to supply `env_var` ourselves.

Anyway, it's most important to be consistent.